### DISCUSSION TITLE
is  @sentry/react-native@6.x.x supported in Expo 51 or 52 beta

### DISCUSSION THREAD

**Discussion by the-simian** [getsentry/sentry-react-native#4216] [Q&A] [answered]
according to `expo-doctor` we still need to downgrade to 5, but it looks like the work for RN 0.76.x is on v6. What's the right version to be using with expo 52 and the "new architecture"?
```
The following packages should be updated for best compatibility with the installed expo version:
  @sentry/react-native@6.1.0 - expected version: ~5.33.1
Your project may not work correctly until you install the expected versions of the packages.
```

**Comment by the-simian** [comment]
```json
  "expo": {
    "install": {
      "exclude": ["@sentry/react-native"]
    },
 }
```

I was able to build successfully for the web target and also report back to sentry when adding this *however* I did experience issues with the `expo-updates` dependency, when adding custom tags and the manifest.

This does seem to, at a high level work with the new architecture. I'm testing devices still

**Reply by the-simian** [reply]
Not marking this as the answer until I test devices and get the expo-updates working. Its just a step in the right direction

**Reply by LunatiqueCoder** [reply]
Hello @the-simian. What are the results of your testing?

 I also had issues for the web with `expo export -p web` because of sentry's config plugin on Expo 52.

**Reply by the-simian** [reply, marked as answer]
@LunatiqueCoder thanks for the reminder. I have "@sentry/react-native": "~6.3.0", working
